setup.py

```diff
@@ -82,6 +82,7 @@
     "typer==0.9.0",
     "colorlog",
     "opacus==1.5.2",
+    "huggingface-hub==0.23.4",
```

Comment:
(nit) latest is 0.25.1

```suggestion
    "huggingface-hub==0.25.1",
```